modules/cluster-logging-collector-external.adoc

```diff
@@ -58,7 +58,7 @@ $ oc edit configmap/fluentd -n openshift-logging
 . Add certificates to be used in `secure-forward.conf` to the existing
 secret that is mounted on the Fluentd pods. The `your_ca_cert` and
 `your_private_key` values must match what is specified in `secure-forward.conf`
-in `configmap/logging-fluentd`:
+in `configmap/logging-collector`:
 +
```

Comment:
in 3.11, this is still called `configmap/logging-fluentd` - in 4.x it is called `configmap/fluentd` - we do not yet have a generic `configmap/collector`

Reply:
@richm Fantastic. Thanks. Find/replace overkill. I will fix these. Thank you!!